tenants/ai-example/single-model-serving/base/main/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+labels:
+- includeSelectors: false
+  pairs:
+    opendatahub.io/dashboard: "true"
+
+resources:
+  - ai-example-single-model-serving-namespace.yaml
```

Comment:
Keep this in the namespaces folder instead of here.

You may end up with multiple apps that deploy to the same namespace so it makes sense to keep it outside of the context of this folder.

Reply:
fixed and moved